README.md

```diff
@@ -23,8 +23,191 @@ Terra Toolkit is a utility module used to facilitate independent development of
+
+  context = 'h2';
+  expect(browser.axe({ viewports, context })).to.be.accessible();
+});
+```
+
+### Visual Regression Tests
+Webdriver.io provides a useful visual regression utility. See [documentation](http://webdriver.io/guide/services/visual-regression.html) for setup instructions. Terra Toolkit provides useful wrappers to make setup easier.
+
+
+
+
+#### Config
+```js
+// wdio.conf.js
+
+// Adds matchReference() chai assertion
+const TerraService = require('terra-toolkit/wdio/services').Terra;
+
+// Use Terra Toolkit visualregression config
+const visualRegression = require('terra-toolkit/wdio/visualcompare');
+
+exports.config = {
+  //...
+  services: [
+    'visual-regression',
+    TerraService
+  ],
+  visualRegression
+  //...
+};
+```
+
+Once added visual regression tests can be written using the commands provided by the [visual regression service](http://webdriver.io/guide/services/visual-regression.html). Example:
+
+```js
+/* global browser, describe, it, expect, viewport */
+describe('button test', () => {
+  // Only test tiny/huge viewports
+  const viewports = viewport('tiny', 'huge');
+  it('checks visual comparison', () => {
+    browser.url('/buttons.html');
+    const screenshots = browser.checkElement('button', { viewports });
+    expect(screenshots).to.matchReference();
+  });
+});
+```
+
+### TerraToolkit service
+
+The terra toolkit webdriver.io service provides custom assertions and globals to make testing easier.
+
+#### Viewports
+
+The `viewports()` function provides a convenience method for fetching an array of viewports to use to resize the browser or even pass directly to the accessibility and visual regression commands.
+
+##### Usage
+wdio-axe-service enhances an WebdriverIO instance with the following command:
+
+`viewports(name);`
+
+The following options are available:
+
+* **name**:
+  Takes the viewport keys and returns an array of `{ height, width}` objects representing the respective viewport size(s). If no key is passed, returns all viewports.
+
+
+##### Example
+
+```js
+/* global browser, describe, it, expect, viewport */
+describe('Resizing browser', () => {
+  // Only test tiny/huge viewports
+  const viewports = viewport('tiny', 'huge');
+  it('switches viewport sizes', () => {
+    browser.url('/foo.html');
+    viewports.forEach(size, () => {
+      // Resize browser to each viewport size
+      browser.setViewportSize(size);
+    });
+  });
+});
+```
+
+#### Assertions
+2 custom assertions are provided to make validating the output of the visual regression and accesibility commands easier.
+
+
+`accessible()`
+
+```js
+// Validate it is accessible
+expect(browser.axe()).to.be.accessible();
+// Validate it is not accessible
+expect(browser.axe()).to.not.be.accessible();
+```
+
+
+`matchReference()`
+
+```js
+// Validate it matches
+expect(screenshots).to.matchReference();
+// Validate it does not match
+expect(screenshots).to.not.matchReference();
+```
+
+
+
+
+## Nightwatch
```

Comment:
Update to `Nightwatch Utility` to keep the table of contents link & summary reference.